content/blog/2020/09/2020-09-09-jenkins-continuous-evolution-cdcon.adoc

```diff
@@ -0,0 +1,40 @@
+
+At link:https://events.linuxfoundation.org/cdcon/[CDCon], on October 7-8, there are two particular Jenkins talks that will focus on new directions that the Jenkins platform is evolving too and getting better and better for users.
+
+== link:https://sched.co/dqOc[Heard of JCasC and Not Sure Where to Start? Let me Help You!]
+Configuration as code is a best practice for your CI/CD setup as it makes the complex process of setting up Jenkins simpler and more reproducible. Jenkins Configuration as Code (JCasc) enables Jenkins users to define the whole configuration as a simple, plain text YAML syntax. With JCasc, setting up a new Jenkins master is easier than ever before. To get started with JCasC some initial effort is required. This talk walks you through a basic setup for easily spinning up new Jenkins instances. 
+
+October 7 at 2:20 PM PST
+Speaker: Ewelina Wilkosz, Eficode
+
+Ewelina W is passionate about making sure that her customers' software is being built, tested and released in the best possible way. And, most importantly, that software developers don’t hate the process. Ewelina has been involved in Jenkins Configuration as Code plugin development from the very beginning. This is a must-see talk where Ewelina will also share some tips and tricks. The talk will feature using Docker, Jenkins and GitHub Actions as a quick way to build… Jenkins! 
+
+== link:https://sched.co/dpvN[Bridging the Gap with Tekton-client-plugin for Jenkins]
+Tekton provides Kubernetes-native CI/CD building blocks. It enables users to take full advantage of cloud-native features around scalability and high availability. Jenkins flexibility enables integration with Tekton. This talk showcases the new tekton-client-plugin for Jenkins that enables Jenkins to interact with Tekton pipelines on a Kubernetes cluster. Tekton and Jenkins are both CDF projects and this talk highlights the first steps towards better Tekton and Jenkins interoperability, a key goal of the CD Foundation.
+
+October 7 at 11:40 AM PST
+Speaker: Vibhav Bobade, Red Hat
+
+=== Register for CDCon
+
+Both these talks showcase the ultimate flexibility and power of the Jenkins platform and how it continues to evolve to meet the challenges of modern-day CI/CD. Don't miss out; link:https://events.linuxfoundation.org/cdcon/[register for CDCon] to attend.
+
+CDCon has pledged to donate 100% of the proceeds received from CDCon 2020 registration to charitable causes: Black Girls Code, Women Who Code and the CDF Diversity Fund. Registrants indicate which charitable fund they want their 25 USD registration fees to go to during registration. If you can’t afford the registration cost, please apply for the link:https://events.linuxfoundation.org/cdcon/attend/scholarships/[diversity scholarship].
+
+image::/images/post-images/2020/cdcon-register.png
```

Comment:
Image is broken (not found in trying to build the page).  Needs to be fixed before the post is merged.
```suggestion
image::/images/post-images/2020/cdcon-register.png
```

Reply:
Fixed in 00cdbcf2744d1334815b419c0f31df7473a1b46a